pages/expandable-section/container-variant.permutations.page.tsx

```diff
@@ -2,28 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0
 import React from 'react';
 import ExpandableSection, { ExpandableSectionProps } from '~components/expandable-section';
-import SpaceBetween from '~components/space-between';
 
 import ScreenshotArea from '../utils/screenshot-area';
-import Link from '~components/link';
-import Button from '~components/button';
 import createPermutations from '../utils/permutations';
 import PermutationsView from '../utils/permutations-view';
+import { headerActions, headerInfo } from './common';
 
 const permutations = createPermutations<ExpandableSectionProps>([
   {
     headerCounter: [undefined, '(5)'],
     headerDescription: [undefined, 'Expandable section header description'],
-    // eslint-disable-next-line react/jsx-key
-    headerInfo: [undefined, <Link variant="info">info</Link>],
-    headerActions: [
-      undefined,
-      // eslint-disable-next-line react/jsx-key
```

Comment:
No, just double-checked that I'm not getting the ESLint warnings that I receive if I remove these ESLint comments in the main branch.

That warning was actually a bit of a weird behavior from ESLint, because to be clear, it was not asking to add a key to the elements _inside_ the SpaceBetween, but to the elements in the `headerInfo` and `headerActions` arrays, that is, to the `Link` and `SpaceBetween` elements themselves —and even to their sibling `undefined` elements (which makes no sense), as if those two arrays would contain a list children to be rendered in a component, though they are not.